src/main/java/edu/kis/powp/jobs2d/builders/Figure2Builder.java

```diff
@@ -0,0 +1,33 @@
+package edu.kis.powp.jobs2d.builders;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.command.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Figures2Builder implements Job2dDriver, Builder {
+    private final List<DriverCommand> commandList = new ArrayList<>();
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.commandList.add(new SetPositionCommand(x, y));
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        this.commandList.add(new OperateToCommand(x, y));
+    }
+
```

Comment:
Code duplication (Figures1Builder is very similar)

Reply:
Removed redundant Figure2Builder.java